src/test/java/org/jenkinsci/main/modules/cli/auth/ssh/CLITest.java

```diff
@@ -118,26 +119,57 @@ private File tempHome() throws IOException {
     @Issue("JENKINS-41745")
     @Test
     public void strictHostKey() throws Exception {
+        Launcher.LocalLauncher localLauncher = new Launcher.LocalLauncher(StreamTaskListener.fromStderr());
+        String jenkinsUrl = r.getURL().toString();
         home = tempHome();
         grabCliJar();
+        String sshCliJar = jar.getAbsolutePath();
 
         r.jenkins.setSecurityRealm(r.createDummySecurityRealm());
         r.jenkins.setAuthorizationStrategy(new MockAuthorizationStrategy().grant(Jenkins.ADMINISTER).everywhere().to("admin"));
         SSHD.get().setPort(0);
         File privkey = tmp.newFile("id_rsa");
+        String keyPath = privkey.getAbsolutePath();
         FileUtils.copyURLToFile(CLITest.class.getResource("id_rsa"), privkey);
-        User.get("admin").addProperty(new UserPropertyImpl(IOUtils.toString(CLITest.class.getResource("id_rsa.pub"))));
-        assertNotEquals(0, new Launcher.LocalLauncher(StreamTaskListener.fromStderr()).launch().cmds(
-            "java", "-Duser.home=" + home, "-jar", jar.getAbsolutePath(), "-s", r.getURL().toString(), "-ssh", "-user", "admin", "-i", privkey.getAbsolutePath(), "-strictHostKey", "who-am-i"
+        User.getById("admin", true).addProperty(new UserPropertyImpl(IOUtils.toString(CLITest.class.getResource(
+                "id_rsa.pub"), UTF_8)));
+
+        assertNotEquals(0, localLauncher.launch().cmds(
+                "java",
+                    "-Duser.home=" + home,
+                    "-jar", sshCliJar,
+                    "-s", jenkinsUrl,
+                    "-ssh",
+                    "-user", "admin",
+                    "-i", keyPath,
+                    "-strictHostKey",
+                    "-logger", "FINE",
+                    "who-am-i"
```

Comment:
Is something actually being changed here? It is hard to tell what you are doing due to the reformatting.

Reply:
no, I simply tried to understand all the parameters and it impossible to make it on a single line.

Reply:
Understood but from the perspective of a reviewer it is hard to look at a long diff containing a lot of no-op hunks and see what is actually changing that needs to be considered. If you want to reformat stuff for general legibility, better to do it as a separate PR marked as being purely stylistic.